src/test/java/org/pesmypetcare/webservice/service/GoogleCalendarServiceTest.java

```diff
@@ -0,0 +1,140 @@
+package org.pesmypetcare.webservice.service;
+
+import com.google.api.client.util.DateTime;
+import com.google.api.services.calendar.model.Event;
+import com.google.api.services.calendar.model.EventDateTime;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import com.google.api.services.calendar.model.Calendar;
+import org.pesmypetcare.webservice.dao.GoogleCalendarDao;
+import org.pesmypetcare.webservice.dao.PetDao;
+import org.pesmypetcare.webservice.entity.EventEntity;
+import org.pesmypetcare.webservice.error.CalendarAccessException;
+import org.pesmypetcare.webservice.error.DatabaseAccessException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.ArgumentMatchers.isNull;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.verify;
+
+/**
+ * @author Marc Simó
+ */
+@ExtendWith(MockitoExtension.class)
+public class GoogleCalendarServiceTest {
+    private static final String CALENDAR_ID_FIELD = "calendarId";
+    private static List<Event> eventList;
+    private static EventEntity eventEntity;
+    private static Event event;
+    private static String accessToken;
+    private static String owner;
+    private static String petName;
+    private static String date;
+    private static String date2;
+    private static String eventId;
+    private static String calendarId;
+
+    @Mock
+    private GoogleCalendarDao googleCalendarDao;
+    @Mock
+    private PetDao petDao;
+
+    @InjectMocks
+    private GoogleCalendarService service = new GoogleCalendarServiceImpl();
+
+    @BeforeAll
+    public static void setUp() {
+        owner = "My owner";
+        petName = "My petName";
+        date = "2020-02-13T10:30:00";
+        date2 = "2020-02-13T12:30:00";
+        eventId = "My eventId";
+        calendarId = "My calendarId";
+        accessToken = "tokem";
+        eventList = new ArrayList<>();
+        eventEntity = new EventEntity();
+        eventEntity.setId("My eventId");
+        eventEntity.setStartDate(new DateTime(date));
+        eventEntity.setEndDate(new DateTime(date2));
+        event = new Event();
+        event.setId("My eventId");
+        EventDateTime start = new EventDateTime()
+            .setDateTime(new DateTime(date))
+            .setTimeZone("Europe/Madrid");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "Europe/Madrid" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5403404)